geowebcache/core/src/main/java/org/geowebcache/storage/CompositeBlobStore.java

```diff
@@ -95,22 +95,26 @@ public LiveStore(BlobStoreConfig config, @Nullable BlobStore store) {
      * on {@link BlobStoreConfig#getId() blobstore id} and TileLayers
      * {@link TileLayer#getBlobStoreId() BlobStoreId} matches.
      * 
-     * @param layers used to get the layer's {@link TileLayer#getBlobStoreId() blobstore id}
-     * @param defaultStorageFinder to resolve the location of the cache directory for the legacy
-     *        blob store when no {@link BlobStoreConfig#isDefault() default blob store} is given
-     * @param configuration the configuration as read from {@code geowebcache.xml} containing the
-     *        configured {@link XMLConfiguration#getBlobStores() blob stores}
-     * @throws ConfigurationException if there's a configuration error like a store confing having
-     *         no id, or two store configs having the same id, or more than one store config being
-     *         marked as the default one, or the default store is not
-     *         {@link BlobStoreConfig#isEnabled() enabled}
-     * @throws StorageException if the live {@code BlobStore} instance can't be
-     *         {@link BlobStoreConfig#createInstance() created} of an enabled
-     *         {@link BlobStoreConfig}
+     * @param layers
+     *            used to get the layer's {@link TileLayer#getBlobStoreId() blobstore id}
```

Comment:
Non-semantic whitespace again.  Yes it does look nicer but it should be done separately.